java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -142,7 +147,7 @@ public MaximumStepsReachedException(String s, TooManyNestedBooleanStatesExceptio
   }
 
   @VisibleForTesting
-  ExplodedGraphWalker() {
+  public ExplodedGraphWalker() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)